src/unit_tests/wazuh_db/CMakeLists.txt

```diff
@@ -19,8 +19,8 @@ list(APPEND wdb_tests_flags "-Wl,--wrap,_mdebug2 -Wl,--wrap,_mdebug1 -Wl,--wrap,
                         -Wl,--wrap,wdbi_query_checksum -Wl,--wrap,wdbi_query_clear -Wl,--wrap,wdb_stmt_cache -Wl,--wrap,wdb_step \
                         -Wl,--wrap,sqlite3_changes -Wl,--wrap,sqlite3_bind_int -Wl,--wrap,sqlite3_bind_text -Wl,--wrap,sqlite3_last_insert_rowid \
                         -Wl,--wrap,sqlite3_step -Wl,--wrap,wdb_open_agent2 -Wl,--wrap,wdb_leave -Wl,--wrap,wdb_agents_insert_vuln_cves \
-                        -Wl,--wrap,sqlite3_errmsg -Wl,--wrap,wdb_agents_clear_vuln_cves -Wl,--wrap,wdb_agents_update_status_vuln_cves \
-                        -Wl,--wrap,wdb_agents_remove_vuln_cves -Wl,--wrap,wdb_agents_remove_by_status_vuln_cves -Wl,--wrap,cJSON_PrintUnformatted \
+                        -Wl,--wrap,sqlite3_errmsg -Wl,--wrap,wdb_agents_clear_vuln_cves -Wl,--wrap,wdb_agents_update_vuln_cves_status \
```

Comment:
Shouldn't it be wdb_agents_update_status_vuln_cves? considering that we have wdb_agents_clear_vuln_cves for example.

Reply:
I followed this convention: `wdb_agents`+ **action**+`vuln_cves`+**extra**.
So we can have clear, insert, update actions; but if we have an extra word like status, it goes at the end.
But it can be changed.

Reply:
I think it's not necessary to follow an order with the TABLE. And the right expression is "_update vuln_cves status_"
Thus, I think it´s good as it is.